rpcs3/rpcs3qt/memory_viewer_panel.cpp

```diff
@@ -272,8 +273,7 @@ void memory_viewer_panel::resizeEvent(QResizeEvent *event)
 	const QMargins margins = layout()->contentsMargins();
 
 	int free_height = event->size().height()
-		- (layout()->count() * (margins.top() + margins.bottom()))
-		- font_height; // bottom margin to allow shrinking
+		- (layout()->count() * (margins.top() + margins.bottom()));
```

Comment:
Add this one line below the for loop below in order to have at least one line of memory at all times:
`setMinimumHeight(event->size().height() - free_height + font_height);`

Resizing to 0 is dangerous and looks dumb !

Reply:
Added.